platform-controller/src/main/java/org/hobbit/controller/docker/ContainerManagerImpl.java

```diff
@@ -372,7 +371,7 @@ private String createContainer(String imageName, String containerType, String pa
         // would like to have OR if there is no parent or the parent is a
         // benchmark (in case of the benchmark controller) and the container has
         // type "system"
-        Integer numberOfSwarmNodes = Integer.MAX_VALUE;
+        long numberOfSwarmNodes = Integer.MAX_VALUE;
```

Comment:
Later in this method we have a branching on `numberOfSwarmNodes > 1`, won't that break it?

Reply:
True. It should be `1` if the cluster manager throws an exception.